backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/datamodels/Rd3Loader.java

```diff
@@ -1,14 +1,13 @@
-package org.molgenis.emx2.datamodels;

```

Comment:
I checkend, this change has to do that it had CRLF newlines. So it is kinda formatting issue